cql/the-basics-of-cql/cinchy-supported-functions/connections-functions.md

```diff
@@ -4,37 +4,79 @@
 
 The set of functions listed in this page are for use in the [Cinchy Connections Experience](https://cli.docs.cinchy.com/) when configuring data syncs.
 
+* [@COLUMN]
 * [​JSON\_ESCAPE​](connections-functions.md#json\_escape)
 * [​URL\_ESCAPE​](connections-functions.md#url\_escape)
 * [STRING\_ESCAPE()](connections-functions.md#string\_escape)
+* [@ID](connections-functions#id)
+
+## @COLUMN
+
+The `@COLUMN` function is used for real-time syncs where your source is an event (such as Cinchy Event Broker) and your target is REST/SOAP. 
+
+Use `@COLUMN` to build the request body of SOAP/REST API specifications.
+
+### Syntax
+
+```
+{
+    "employeeNane": "@COLUMN('Name')"
+}
+```
+
+### Example
+
+The following example uses the Cinchy Event Broker as a Source with a REST API destination. 
+
+This scenario maps the data from the `Name` property to the REST API destination property `employeeName`.
+
+1. In the **Source** tab, set the source to Cinchy Event Broker. 
+2. In the **Schema** section, select your schema. In this example, the schema consists of two columns: 
+
+ - `Id`, with a **Data Type** of Number.
+ - `Name` with a **Data Type** of Text.
+
+2. In the **Destination** tab, go to **REST API** > **API Specification** > **Insert Specification** > **Request** and select **POST** with an endpoint URL.
+
+
+3. Enter your mapping into the **Body**. This scenario uses the following:
+
+    ```json
+    {
+        "employeeNane": "@COLUMN('Name')",
+        "id": "@ID"
+
+
+    }
+    ```
 
 ## JSON\_ESCAPE <a href="#json_escape" id="json_escape"></a>
 
 This function is used in Connections to escape parameter values and be safe to use inside a JSON document without breaking it
```

Comment:
Breaking what? the JSON document or the parameter values?

Better(?): This function escapes special characters in parameter values, ensuring they can be safely embedded within a JSON document without causing syntax errors.